tables/hbo.tbl

```diff
@@ -0,0 +1,263 @@
+#However, Classical or Biblical Hebrew, which is used in the Hebrew Bible and other liturgical literature, contains cantillation marks that are not supported by the Hebrew table mentioned above. This table seeks to provide access to these accents. This table is   also based upon the tables mentioned above.  This table only departs from it where the accents are concerned. With this table, the user will have access to the accents and Masorah in BHS and BHS Quinta. For more information about the IHBC and How it was developed:
+#https://en.wikipedia.org/wiki/Hebrew_Braille#:~:text=The%20International%20Hebrew%20Braille%20Code
+#also see: 
+#https://huc.edu/library_blog/learning-about-hebrew-braille/#:~:text=The%20Code%20created%20by%20Brevis,books%20written%20in%20Hebrew%20Braille.
+
+#-maintainer: Paul Geoghegan <contact@envisionly.tech>
+#-maintainer: Matityhau Yeshurun <yeatersink@gmail.com>
+letter שְּׁ 146-3
+letter שְּׂ 5-156-3
+letter פְּ 5-1234-3
+letter תְּ 2345-3
+letter מְּ 5-134-3
+letter כְּ 13-3
+letter נְּ 5-1345-3
+letter יְּ 5-245-3
+letter קְּ 5-12345-3
+letter ּוְ 136-3
+letter דְּ 5-145-3
+letter שְׁ 146-3
+letter בְּ 12-3
+letter שּׁ 146-3
+letter שּׂ 5-156
+letter שְׂ 156-3
+letter לְּ 5-123-3
+letter ש 146
+letter הּ 125
+letter קּ 5-12345
+letter ר 1235
+letter וֹ 246
+letter וּ 136
+letter ּו 136
+letter ֵי 3456
+letter יּ 5-245
+letter שׁ 146
+letter פּ 5-1234
+letter ִי 35
+letter כּ 13
+letter נּ 5-1345
+letter כְּ 13-3
+letter לּ 5-123
+letter דּ 5-145
+letter בּ 12
+letter בְּ 12-3
+letter תְּ 5-1456-3
+letter מּ 5-134
+letter שׂ 156
+letter תּ 2345
+letter ק 12345
+sign ✙ 23456-6
+letter צ 2346
+sign ❑ 12345
+sign ✔ 12356
+sign ❦ 6-156
+sign ✗ 24
+sign ❥ 16-6
+sign ✘ 35
+letter ץ 2346
+sign ❚ 14
+sign ❁ 123-1346-1346
+sign ❘ 1256
+sign ❍ 135
+sign ❏ 135-6
+sign ❙ 2345
+sign ❐ 1235
+sign ✕ 1235
+sign ✖ 2456
+sign ❧ 156-6
+letter פ 1234
+letter ם 134
+letter ֤ 46-125
+letter ֫ 46-135
+letter ֪ 46-13456
+letter ֩ 46-12345
+letter ֨ 46-1234
+letter ֧ 46-145
+letter ֦ 46-13
+letter ֥ 46-134
+letter ֣ 46-136
+letter ֭ 46-24
+letter ֢ 46-23
+letter ֡ 46-236
+letter ֠ 46-2356
+letter ֟ 46-1345
+letter ֞ 46-12
+letter ֝ 46-16
+letter ֜ 46-1
+letter ֛ 46-1236
+letter ֬ 46-24
+letter ֮ 46-1356
+letter ל 123
+letter ֹ 246
+letter ֲ 25
+letter ֳ 345
+letter ִ 24
+letter ֵ 34
+letter ֶ 15
+letter ַ 14
+letter ָ 126
+letter ֺ 5-246
+letter ֯ 456-134
+letter ֻ 346
+letter ּ 5
+letter ֽ 4
+punctuation ־ 36
+letter ֿ 2
+punctuation ׀ 456
+letter ׁ 4
+letter ֚ 46-245
+letter ְ 3
+letter א 1
+letter י 245
+letter ט 2345
+letter ח 1346
+letter ז 1356
+letter ו 2456
+letter ה 125
+letter ד 145
+letter ג 1245
+letter ב 1236
+letter בּ 12
+letter ף 124
+punctuation ״ 456
+letter ׳ 456-126
+letter ׇ 126
+letter ׆ 456-1345
+letter ׅ 45
+letter ׄ 4
+letter ׃ 256
+letter ת 1456
+letter תּ 5-1456
+letter ׂ 45
```

Comment:
i thought this letter was a fleck of dust on my screen but i zoomed in and it looks like it's a little high-up dot (maybe holam, "o"?). i noticed that it's being given the same braille character, 45, as the little low-down dot on line 157 (which might be a khirik, which I see you have defined on line 125 as 24). if this one is a holam, should it perhaps be 246? that's what wikipedia thinks it should be: https://en.wikipedia.org/wiki/Hebrew_Braille

then there's another little high-up dot on line 158 that's braille character 4 - is it a shuruk or a dagesh, or maybe a shin dot or a sin dot? i don't know what it is, but this seems like a good opportunity to use comments to clarify which dots these are.